src/crl/interactivesessions/shells/bashshell.py

```diff
@@ -77,13 +77,13 @@ def _set_bash_environment(self):
         logger.log(_LOGLEVEL, "Preparing bash session")
         output = self._read_until_end()
 
-        self._send_input_line('unset HISTFILE')
+        self._send_input_line('unset HISTFILE', remove_from_buffer=False)
```

Comment:
My guess is that the actual problem is in self._read_until_end() call which is not reading the message of day properly. The second problem is that we do not actually know what is the `termios` echo input line state prior this. Hard coding this to via `remove_from_buffer=False` is rather bad idea as then we state that surely echo is off.